### DISCUSSION TITLE
tohum anahtarla ilgili sorun

### DISCUSSION THREAD

**Discussion by SerkaanD** [IBAX-io/go-ibax#1917] [Bug Submission]
cüzdanımı başka bir bilgisayarda açmak için tohum kelimeleri yükleyince hata alıyorum. olmuyor. ciddi bir hata olduğunu düşünüyorum paramızı cüzdana aktaracağız sonuçta bilgisayar çökse tüm paramız yok olacak

**Comment by Omrumbenim** [comment]
???
